src/accessories/CoverAccessory.ts

```diff
@@ -10,6 +10,10 @@ import { BaseAccessory } from "./BaseAccessory";
 import { TuyaDevice } from "../api/response";
 
 export class CoverAccessory extends BaseAccessory {
+  public target = 0;
+  public position = 0;
+  public motor = 2;
```

Comment:
Please change these [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) to enums

That will also possibly explain why this value defaults to 2 and the others default to 0